receiver/redisreceiver/proto.go

```diff
@@ -24,11 +24,11 @@ import (
 
 // Helper functions that produce protobuf
 
-func newMetricsData(protoMetrics []*metricspb.Metric) *consumerdata.MetricsData {
+func newMetricsData(protoMetrics []*metricspb.Metric, serviceName string) *consumerdata.MetricsData {
 	return &consumerdata.MetricsData{
 		Resource: &resourcepb.Resource{
 			Type:   typeStr,
-			Labels: map[string]string{"type": typeStr},
+			Labels: map[string]string{"service.name": serviceName},
```

Comment:
Probably you should add this only if not empty?

Reply:
Yes I can do that. I was thinking this would be a required field (I put that in the docs, too, but can take it out).

Reply:
Ok, we need to revisit this anyway, as I mentioned I think we should let user define more attributes in the `Resource` not just this one.